eslint.config.mjs

```diff
@@ -16,29 +18,25 @@ export default [
         name: "eslint/global-rules",
         languageOptions: {
             "ecmaVersion" : 2020,
```

Comment:
any reason why you replace the previous emacVersion 2022 by 2020 ?
I did changed back to 2022 while resolving conflicts, but if there is an actual solid reason, happy to put back 2020

Reply:
Not sure why I changed it. Good you caught that 👍